src/wcmCommon.c

```diff
@@ -625,6 +625,30 @@ static void sendCommonEvents(WacomDevicePtr priv, const WacomDeviceState* ds,
 		sendWheelStripEvents(priv, ds, first_val, num_vals, valuators);
 }
 
+/* 1:1 copy from xf86ScaleAxis */
+int wcmScaleAxis(int Cx, int to_max, int to_min, int from_max, int from_min)
+{
+	int X;
+	int64_t to_width = to_max - to_min;
+	int64_t from_width = from_max - from_min;
+
+	if (from_width) {
+		X = (int)(((to_width * (Cx - from_min)) / from_width) + to_min);
+	}
+	else {
+		X = 0;
+		/*ErrorF ("Divide by Zero in xf86ScaleAxis\n");*/
```

Comment:
tbh, if this ever happens in our code we've screwed up several layers of checks. That error message makes sense in the server where it's a more generic API but there is no way where our driver would allow a zero-width.